website/docs/developer/research/research-localstack.md

```diff
@@ -257,23 +285,26 @@ Some examples of CLI commands and their corresponding outputs are shown below.
             }
         ]
     }
+```
 
+#### Secrets manager
+```commandline
+  awslocal secretsmanager create-secret --name localstack-secret
+```
```

Comment:
See notes above about combining the code blocks and adding some comments for users.